spec/grape/api_spec.rb

```diff
@@ -2196,8 +2196,6 @@ def foo
     it 'uses default_rescue_handler to handle invalid response from rescue_from' do
```

Comment:
I suggest to change the name of the example to reflect that it doesn’t anymore test the implementation detail that it uses `default_rescue_handler`.

Reply:
I made a change